stl/inc/xtr1common

```diff
@@ -165,6 +165,12 @@ template <class _Ty, class... _Types>
 _INLINE_VAR constexpr bool _Is_any_of_v = // true if and only if _Ty is in _Types
     disjunction_v<is_same<_Ty, _Types>...>;
 
+#ifndef __CUDACC__
+_NODISCARD constexpr bool _Is_constant_evaluated() noexcept { // Internal function for any standard mode
+    return __builtin_is_constant_evaluated();
+}
+#endif // __CUDACC__

```

Comment:
It was necessary when `__builtin_is_constant_evaluated()` was not defined, apparently it is no longer neccessary as well as `&& !defined(__CUDACC__)` after `_USE_STD_VECTOR_ALGORITHMS`

Reply:
Great! In order to avoid breaking the Intel compiler gratuitously, I also checked them, and it appears that their latest versions support it too: https://godbolt.org/z/bdfPxsrdb

I will validate and push changes.